README.md

```diff
@@ -16,6 +16,13 @@ The project is currently being hosted on **Heroku** cause it's cheap and easy bu
 ## Installation
 In order to run the project locally you'll need `node` installed as well as `nvm`.
 
+Install [Homebrew](https://brew.sh/)
+`/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"`
+
+Follow instructions [here](http://blog.teamtreehouse.com/install-node-js-npm-mac) to install `node` and `nvm`
+`brew install node`
```

Comment:
these two lines combine them into one big backtick with the sentences on different lines  instead of 2 separate back ticks